sdk/communication/AzureCommunicationCommon/CHANGELOG.md

```diff
@@ -1,11 +1,8 @@
 # Release History
 
-## 1.3.0 (upcoming)
+## 1.2.0 (upcoming)
```

Comment:
When is the supposed to be released? It shouldn't say "upcoming"

Reply:
Hi Travis! We will create another release PR to update the date and version in required files